Document use of "chef-client -j file.json" passed attributes (#2201)
chef/chef-web-docs: chef_master/source/ctl_chef_client.rst

   The frequency (in seconds) at which Chef Infra Client runs. When running Chef Infra Client at intervals, apply ``--splay`` and ``--interval`` values before a Chef Infra Client run. Default value: ``1800``.

``-j PATH``, ``--json-attributes PATH``

>    The path to a file that contains JSON data. Used to setup the first client run. **For all the future runs with option -j the attributes are expected to be persisted in the chef-server.**

Reviewer: ```suggestion
   The path to a file that contains JSON data. Used to setup the first client run. The attributes will persist on the Chef Infra Server for all future runs with option ``-j``.
```
